src/spacel/provision/template/app.py

```diff
@@ -141,6 +143,22 @@ def app(self, app_region):
                 }
             }
 
+        # Custom update policy:
+        if app_region.update_policy == 'disabled':
+            del resources['Asg']['UpdatePolicy']
+        if app_region.update_policy == 'redblack':
+            resources['Asg']['UpdatePolicy'] = {
+                'AutoScalingReplacingUpdate': {
+                    'WillReplace': 'true'
+                }
+            }
+            time_pos = (resources['Lc']
```

Comment:
Perhaps if we run a `changeset` and don't detect any changes, we can force a rotate?